control-plane-operator/controllers/hostedcontrolplane/kas/deployment.go

```diff
@@ -438,6 +438,20 @@ func buildKASContainerMain(image string, port int32, noProxyCIDRs []string) func
 		// https://github.com/kubernetes/kubernetes/blob/ab13c85316015cf9f115e29923ba9740bd1564fd/staging/src/k8s.io/apimachinery/pkg/util/net/http.go#L112-L114
 		proxy.SetEnvVars(&c.Env, noProxyCIDRs...)
 
+		if hcp.Annotations[hyperv1.KubeAPIServerGOGCAnnotation] != "" {
```

Comment:
Do these need any type of validation? What happens if an invalid value is used, would Go ignore the environment variable?

Reply:
The env vars are ignored if they don't have a valid value